blogs/AIB/aib.md

```diff
@@ -4,130 +4,91 @@
 
 Conversely, AI can reciprocate by improving the architecture of blockchain systems, making them more secure, energy-efficient, and effective. In essence, the convergence of AI and blockchain technologies results in an autonomous, credible, and intelligent system, leveraging the benefits of both while mitigating their respective shortcomings.
- 
-
-## Application Scenario
 
-In this section, we will discuss the combination of AI and blockchain technology in detail. At the 2020 International Conference on Information Systems, Erik Karger proposed a detailed review on the combination of blockchain and AI {cite}`inproceedings`. According to his research, the integration of AI and blockchain can be categorized into three main areas, each with its own subdivisions.
+## Enhancing DeFi Operations with AI
 
-```{note} **AI & Blockchain Combination Classification**
+Most commonly, research and applications have explored the potential of using blockchain to augment AI {cite}`inproceedings`. However, our focus is on exploring how AI can enhencing Decentralized Finance. As a core application of blockchain, Decentralised Finance (DeFi) has consistently been a subject of keen interest in both academic and commercial research. 
 
-- Blockchain for AI: Blockchain for AI pertains to scenarios and applications in which blockchain technology is utilized to bolster or amplify pre-existing AI applications.
-- AI for Blockchain: AI for Blockchain encompasses situations where AI applications strive to enhance blockchain technology. This can be seen in areas like mining processes and smart contract execution.
-- AI with Blockchain: AI with Blockchain refers to systems and platforms that concurrently employ AI and blockchain technology, with neither being solely reliant on or used as a leveraging tool for the other.
+Raheman et al. cite{`9686345`} designed an infrastructure of AI agents or "Oracles" for portfolio management, liquidity provision, and price prediction in various decentralized financial markets. As shown in Figure 1, these Oracles will increase investment value and returns by offering liquidity. They serve end business applications, smart contracts, and other agents. A key aspect is the distinction between an "inventory/portfolio" that comprises multiple assets and a "DEX swap pool" or "DEX balancing pool," which is one of several portfolio maintenance strategies. Therefore, a single "inventory/portfolio" may contain multiple "DEX swap pools" or "DEX balancing pools" with various strategies.
 
-```
-```{figure} images/AIB.png
+```{figure} images/hashgraph.png
 ---
-width: 550px
-height: 300px
-name: AIB-fig
+width: 450px
```

Comment:
the ratio makes the image look squeezed -- and the caption has "Figure 1" hardcoded @yimikaerinle @aik31 

<img width="869" alt="image" src="https://github.com/dlt-science/science-notes/assets/31636867/f51ca2e6-87e7-4696-8fe1-8395153d92f2">